assets/sass/components/_link-embedder.scss

```diff
@@ -15,7 +15,7 @@
 
 .embed iframe {
   max-width: 90vw;
-  max-height: (344 / 459 * 90vw);
+  max-height: (0.7494553377 * 90vw) // 344 / 459 = 0.7494553377
```

Comment:
Sass has deprecated "/" as division. Need to use math.div() instead, but multiply works well enough.